framework/include/fvbcs/FVDirichletBCBase.h

```diff
@@ -0,0 +1,31 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "FVBoundaryCondition.h"
+
+/**
+ * Base class for finite volume Dirichlet boundaray conditions. A base class is
+ * needed because Dirichlet BCs must be handled specially when computing fluxes
+ * on faces in the FV method.
+ */
+class FVDirichletBCBase : public FVBoundaryCondition
+{
+public:
+  /**
+   * Class constructor.
```

Comment:
comments on constructors that say "constructor" are... well I don't like them.

Reply:
Agreed. But honestly I don't think a constructor should require documentation unless it does something unexpected. Can I totally remove this doxygen block or do our code standards require that we have one?

Reply:
I'm okay with you killing it.